src/Connection.php

```diff
@@ -45,6 +45,8 @@
  *
  * @psalm-import-type Params from DriverManager
  * @psalm-consistent-constructor
+ * @phpstan-import-type ParameterTypeOfValue from ParameterType
+ * @psalm-import-type ParameterTypeOfValue from ParameterType
```

Comment:
You can also remove all the double imports I think (keep the Psalm version please)

Reply:
I might be wrong of course so please test before doing it everywhere.

Reply:
Yes, PHPStan also understand `psalm-import-type`